### PR TITLE
add trec-covid pre-built index to pyserini

### PR DESCRIPTION
**Pull request by yuki617**
Add trec-covid round1-5  pre-built index

### CODE REVIEW

File: pyserini/prebuilt_index_info.py [hunk 1]
```diff
@@ -151,5 +151,65 @@
...
+    },
+    "trec-covid5": {
+        "description": "Trec Covid Round 5",
+        "urls": [
+            "https://www.dropbox.com/s/9hfowxi7zenuaay/lucene-index-cord19-abstract-2020-07-16.tar.gz?dl=1"
+        ],
+        "md5": "c883571ccc78b4c2ce05b41eb07f5405",
+        "size compressed (bytes)": 2796524,
+        "total_terms": -1,
+        "documents": 18,
```
Comment (lintool) [inline pyserini/prebuilt_index_info.py:163]: these stats don't seem right?

Comment (lintool) [reply pyserini/prebuilt_index_info.py:163]: This method should check out:
https://github.com/castorini/pyserini/blob/master/pyserini/index/_base.py#L180

**Review by lintool — CHANGES_REQUESTED** [review]
Also, mirror in gitlab.

File: pyserini/prebuilt_index_info.py (hunk 1 above)
Comment (yuki617) [reply pyserini/prebuilt_index_info.py]: Sorry, the number should be fixed now.

File: pyserini/prebuilt_index_info.py [hunk 2]
```diff
@@ -151,5 +151,65 @@
         "documents": 4170312,
         "unique_terms": -1,
         "downloaded": False
+    },
+    "trec-covid5": {
```
Comment (lintool) [inline pyserini/prebuilt_index_info.py:155]: Please change to `trec-covid-r5-abstract`
Description should be "TREC-COVID Round 5: abstract index"
Add gitlab link here: https://git.uwaterloo.ca/jimmylin/cord19-indexes

We should also have:
+`trec-covid-r5-full-text` description: "TREC-COVID Round 5: full-text index"
+`trec-covid-r5-paragraph` description: "TREC-COVID Round 5: paragraph index"

File: pyserini/prebuilt_index_info.py [hunk 3]
```diff
@@ -151,5 +151,65 @@
...
+            "https://www.dropbox.com/s/9hfowxi7zenuaay/lucene-index-cord19-abstract-2020-07-16.tar.gz?dl=1"
+        ],
+        "md5": "c883571ccc78b4c2ce05b41eb07f5405",
+        "size compressed (bytes)": 2796524,
+        "total_terms": 22100404,
+        "documents": 192459,
+        "unique_terms": 195875,
+        "downloaded": False
+    },
+    "trec-covid4": {
```
Comment (lintool) [inline pyserini/prebuilt_index_info.py:167]: For all rounds, we should have three separate indexes, like above.

**Review by lintool — CHANGES_REQUESTED** [review]

**Comment by lintool** [comment]
Please add additional indexes - per comments.

File: pyserini/prebuilt_index_info.py (hunk 3 above)
Comment (yuki617) [reply pyserini/prebuilt_index_info.py]: All full-text and paragraph pre-built index are added

**Comment by lintool** [comment]
Gitlab URLs need to be changed to this form `https://git.uwaterloo.ca/jimmylin/anserini-indexes/raw/master/index-robust04-20191213.tar.gz` so that we're downloading the raw tarball, not the HTML landing page.

**Comment by yuki617** [comment]
> the raw tarball, not the HTML landing page.
Thanks for catching that. The urls should be the raw tarball ones now

**Review by lintool — APPROVED** [review]

**Pull request merged**